cmd/utils.go

```diff
@@ -44,6 +44,10 @@ func preRunConfig(cmd *cobra.Command, args []string) {
 	// Set up scorecard wait time default
 	viper.SetDefault("scorecard_wait_time", DefaultScorecardWaitTime)
 
+	// Set up pyxis host
+	viper.SetDefault("pyxis_host", DefaultPyxisHost)
+	viper.SetDefault("pyxis_api_token", "")
```

Comment:
Is this just temporary? We are taking this value in as an argument, do we need to expose it as an arg and an env?

Reply:
I'd like to see both tbh. Then the secret (api token) can be injected as an env var and not have to specify on the command line.

Reply:
That makes sense and probably provides better usability if someone is running this over and over again before `submit`.